src/ascend.js

```diff
@@ -0,0 +1,29 @@
+var _curry1 = require('./internal/_curry1');
+
+
+/**
+ * Makes an "ascending" comparator function out of a function that returns a value
+ * that can be compared
+ *
+ * @func
+ * @memberOf R
+ * @since v0.23.0
```

Comment:
You do not need to include these, `@since` annotations [are added automatically](https://github.com/ramda/ramda/blob/master/CONTRIBUTING.md)